### ISSUE TITLE
enhancement: slider to browse through large books

### ISSUE THREAD

**Issue by hackerb9** [SufficientlySecure/document-viewer#150]
When reading a large book or using single page view it can be awkward to quickly jump to a faraway page. It would be nice if there was a horizontal slider that let a person flip quickly to at least the general area. It'd be especially nice if the slider had detents marking chapters. I imagine it being enabled the same as the zoom wheel. This would also satisfy readers who like to have a progress bar as they read.

**Comment by ericwa** [comment]
Yes, agree this would be good to have. There is a slider in the "Go to..." dialog, but it's not that useful because you can't see the pages as you move the slider.
